tools/block_cache_analyzer/block_cache_trace_analyzer.cc

```diff
@@ -579,8 +579,8 @@ void BlockCacheTraceAnalyzer::WriteSkewness(
   // Sort in descending order.
   sort(
       pairs.begin(), pairs.end(),
-      [=](std::pair<std::string, uint64_t>& a,
-          std::pair<std::string, uint64_t>& b) { return b.second < a.second; });
+      [=](const std::pair<std::string, uint64_t>& a,
+          const std::pair<std::string, uint64_t>& b) { return b.second < a.second; });
```

Comment:
The body of the lambda should be in a separate line, according to `make format`.

Reply:
Done